vm/tests/grammar.rs

```diff
@@ -715,6 +715,24 @@ fn peek_all() {
     };
 }
 
+#[test]
+fn peek_slice_23() {
```

Comment:
This test should be in `derive/tests/grammar.rs` as well.

Reply:
done!